### PR TITLE
Update SDK to 7.4.1

### PR DESCRIPTION
**Pull request by etharner**
For @thepiwo: it also consist of fixes required for account switching, please check.

### CODE REVIEW

File: src/store/plugins/initSdk/index.js [hunk 1]
```diff
@@ -142,13 +142,25 @@ export default (store) => {
           }),
         }],
         compilerUrl: network.compilerUrl,
+        accounts: store.state.accounts.list
+          .reduce((a, { address }) => ([{
+            sign: () => {},
+            address: () => new Promise(resolve => resolve(address)),
+          }, ...a,
+          ]), []),
```
Comment (davidyuk) [inline src/store/plugins/initSdk/index.js:150]: can it be replaced by `map` + maybe `reverse`?

**Comment by thepiwo** [comment]
> For @thepiwo: it also consist of fixes required for account switching, please check.

works for me, thanks

File: src/store/plugins/initSdk/index.js [hunk 2]
```diff
@@ -236,6 +236,7 @@ export default (store) => {
       if (isEqual(currentNetwork, lastNetwork)) return;
       lastNetwork = currentNetwork;
       await recreateSdk();
+      store.state.sdk.selectNode(currentNetwork.name);
```
Comment (davidyuk) [inline src/store/plugins/initSdk/index.js:239]: Why don't we make it a part of `createSdk`? Also, is it true that sdk doesn't select node by default?

Comment (etharner) [reply src/store/plugins/initSdk/index.js]: It does, but sdk calls `selectNode` from `node-pool`, not from `wallet-rpc`, but notification about changing the node (which leads to `onNetworkChange` in aepp) comes only from wallet-rpc's `selectNode` (and it calls node-pool's `selectNode` btw)
https://github.com/aeternity/aepp-sdk-js/blob/23de7edfacc5830b987c868b4525d8de3860b6fa/es/utils/aepp-wallet-communication/rpc/wallet-rpc.js#L272-L280

**Pull request merged**